### ISSUE TITLE
JSONP error handler for cross origin requests is triggered only after timeout

### ISSUE THREAD

**Issue by mzgol** [jquery/jquery#1743]
Originally reported by ri.joel@… at: http://bugs.jquery.com/ticket/13806
<p>
The JSONP error handler is triggered for HTTP errors (e.g. 404) for same origin requests immediately, but for cross origin requests only after the timeout occurred. This can be easily fixed with adding script.onerror (see Zepto.js).
</p>
<p>
<a class="ext-link" href="http://jsfiddle.net/MFmnu/" rel="nofollow"><span class="icon">&nbsp;</span>http://jsfiddle.net/MFmnu/</a>
</p>
<p>
Tested browsers:
Affected: Chrome 26, Firefox 20, Safari 6, Opera 12.14
Not affected: IE 9
</p>
<p>
I will provide a patch later.
</p><p>Issue reported for jQuery 1.9.1</p>

**Comment by mzgol** [comment]
Since we're not going to release the Compat line after all, I'm closing this as "Won't fix".
